upstream-community-operators/kiali/kiali.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiali-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: Observability
```

Comment:
Please use categories from the following list: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories

Reply:
can I have multiple categories here? This would be appropriate for kiali:

```
categories:
- Monitoring
- Logging & Tracing
```

Reply:
I changed the categories to what my previous comment mentions. I think that is Ok.

Reply:
This should be a comma-separated String of categories

Reply:
now using

```
categories: Monitoring,Logging & Tracing
```